.k8s/live/dev/ingress.yaml

```diff
@@ -3,7 +3,11 @@ kind: Ingress
 metadata:
   name: laa-court-data-ui-app-ingress
   namespace: laa-court-data-ui-dev
+  labels:
+    app.kubernetes.io/managed-by: "Helm"
   annotations:
+    meta.helm.sh/release-name: "laa-court-data-ui"
+    meta.helm.sh/release-namespace: "laa-court-data-ui-uat"
```

Comment:
this is Dev ingress but the namespace says UAT

Reply:
maybe I am missing the point here 🤔 
what is the logic that maps the ingress to the release-namespace?

Reply:
Arg! No I messed up. Thanks for spotting.

Reply:
Now fixed.